core/gui/src/app/workspace/component/result-panel/result-panel.component.ts

```diff
@@ -40,6 +41,9 @@ export class ResultPanelComponent implements OnInit, OnDestroy {
   operatorTitle = "";
   dragPosition = { x: 0, y: 0 };
   returnPosition = { x: 0, y: 0 };
+  isDocked = true;
+  undockedResizeDirections = ["right", "bottom", "bottomRight"];
```

Comment:
since they are all constants, no need to save them as variables.

Reply:
changed